dotnet/src/Extensions/PromptTemplate.Handlebars/HandlebarsPromptTemplateOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using HandlebarsDotNet;
+using HandlebarsDotNet.Helpers.Enums;
+using HandlebarsDotNet.Helpers.Options;
+
+namespace Microsoft.SemanticKernel.PromptTemplates.Handlebars;
+
+/// <summary>
+/// Configuration for Handlebars helpers.
+/// </summary>
+public sealed class HandlebarsPromptTemplateOptions : HandlebarsHelpersOptions
+{
+    // TODO [@teresaqhoang]: Issue #3947 Add Categories filter for KernelSystemHelpers (i.e., KernelHelperCategories)
+
+    /// <summary>
+    /// Default character to use for delimiting plugin name and function name in a Handlebars template.
+    /// </summary>
+    public string DefaultNameDelimiter { get; set; } = "-";
```

Comment:
Is this intended to be an instance member rather than static? This appears to only be read in the ctor, which will happen before someone gets a chance to set this.  Is this even needed?

Reply:
No, I removed it altogether, set `"-"` as the default delimiter in the constructor